lib/ffi/struct.rb

```diff
@@ -341,7 +341,7 @@ def find_type(type, mod = enclosing_module)
       # @raise if Ruby 1.8
       # Add hash +spec+ to +builder+.
       def hash_layout(builder, spec)
-        raise "Ruby version not supported" if RUBY_VERSION =~ /1\.8\.*/
```

Comment:
Please decouple from rubygems. Correcting the regex to `/^1\.8\..*/` is probably a better route.